gen/schema/schema.yaml

```diff
@@ -48,6 +48,7 @@ attribute:
   string_patterns: list(str(), required=False) # List of regular expressions that the string must match, only relevant if type is "String"
   string_min_length: int(required=False) # Minimum length of a string, only relevant if type is "String"
   string_max_length: int(required=False) # Maximum length of a string, only relevant if type is "String"
+  read_only: any(str(), int(), bool(), required=False) # Sets field as read-only (computed) with the provided value. This value may be overriden by value returned from API
```

Comment:
Could we rename this to `computed` and make it boolean to align with what we already have for ISE? I don't think we need the expected value here as it should always be retrieved from the API response.

Reply:
changed